astacus/coordinator/api.py

```diff
@@ -81,6 +82,12 @@ async def backup(*, c: Coordinator = Depends(), op: BackupOp = Depends(BackupOp.
     return c.start_op(op_name=OpName.backup, op=op, fun=runner)
 
 
+@router.post("/deltabackup")
```

Comment:
For consistency with /delta/list and the same pattern used in node ops, this could be better as `/delta/backup`

Reply:
the routing and result routing was built with the assumption that the operation name is a single word. I cheated a bit and reused the "backup" op type, should be good enough.